app/src/main/java/fr/free/nrw/commons/media/CommonsWikibaseItem.java

```diff
@@ -0,0 +1,64 @@
+package fr.free.nrw.commons.media;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.Map;
+
+
+/**
+ * Represents the Wikibase item associated with a Wikimedia Commons file.
+ * For instance the Wikibase item M63996 represents the Commons file "Paul Cézanne - The Pigeon Tower at Bellevue - 1936.19 - Cleveland Museum of Art.jpg"
+ */
+public class CommonsWikibaseItem {
+
+    @SerializedName("type")
```

Comment:
javadoc for all fields, with example values

Reply:
added